themes/finna2/templates/Helpers/holding-callnumber.phtml

```diff
@@ -23,5 +23,8 @@
       </ul>
     </div>
   <?php endif; ?>
+  <?php if ($this->wayfinder): ?>
+    <div class="wayfinder-placeholder" data-location='<?=$this->wayfinder?>'></div>
```

Comment:
There are situations where simple `json_encode` is not enough to safely encode JSON in an HTML attribute. I'd suggest making `$this->wayfinder` an array and encoding it here with `$this->HtmlSafeJsonEncode()`.